packages/core/package.json

```diff
@@ -8,26 +8,38 @@
   "scripts": {
     "lint": "eslint \"**/*.ts*\"",
     "build": "rm -rf dist && tsup index.ts",
+    "test": "jest",
```

Comment:
Oops, we need to add this to the root `package.json` (`turbo run test`) for the CI script to work! Would you add mind adding that too, please?

Reply:
You'll also need to add this to `turbo.json`

```
"test": {},
```